auth/tests/unit/token_test.py

```diff
@@ -1,5 +1,33 @@
-import gcloud.aio.auth.token as token  # pylint: disable=unused-import
+import io
+import json
+
+import gcloud.aio.auth.token as token
+
+import pytest
 
 
 def test_importable():
     assert True
+
+
+@pytest.mark.asyncio
+async def test_service_as_io():
+    service_data = {
```

Comment:
You can go ahead and insert a `# pylint: disable=line-too-long` at the top of this method, since that's totally ok for test data; but please fix the other lint issue (4-space indents instead of 2-space)